LinkBase/LinkBase/CommsQuestion.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CommsQuestion.swift
+//  LinkBase
+//
+//  Created by Barbara Ristau on 3/27/17.
+//  Copyright © 2017 barbrobmich. All rights reserved.
+//
+
+import Foundation
+
+struct CommsQuestion {
+    
+    var question: String?
+    var questionID: Int?
+    
+    init() {
+        question = "Why did you decide to study computer science?"
```

Comment:
Yes, that's exactly correct.  I'm investigating how to best set up the model.  It should be an array of questions, and then in the view controller, I would like the question to be randomly generated.  So the question that appears right now is a placeholder, but want to make sure we could get the model linked to the vc.